src/runtime/google-analytics-event-listener.js

```diff
@@ -40,8 +41,9 @@ export class GoogleAnalyticsEventListener {
   /**
    *  Listens for new events from the events manager and logs appropriate events to Google Analytics.
    * @param {!../api/client-event-manager-api.ClientEvent} event
+   * @param {(!../api/client-event-manager-api.ClientEventParams|undefined)=} analyticsParams
    */
-  handleClientEvent_(event) {
+  handleClientEvent_(event, analyticsParams = undefined) {
```

Comment:
Should this be `eventParams` and
`const analyticsParams = eventParams.googleAnalyticsParameters;` ?

Reply:
good catch!